libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -229,9 +229,9 @@ export const sparkpostConfig: IConfigCredentials[] = [
   },
   {
     key: CredentialsKeyEnum.Region,
-    displayName: 'EU',
+    displayName: 'Region',
     description: 'Use `eu` if your account is registered to SparkPost EU',
-    type: 'boolean',
+    type: 'string',
```

Comment:
This is the point that I mention it was going to be tricky. But I didn't explain the things properly. I will follow up in the thread of the comment.